partner_duplicate_mgmt/wizard/base_partner_merge_automatic_wizard.py

```diff
@@ -52,13 +69,22 @@ def _update_children(self, src_partners, dst_partner):
             'parent_id': dst_partner.id,
         })
 
+    def _check_partner_not_merged_with_parent(self, partners):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/odoo-partner-addons/pullRequest?prid=3780056)